packages/components/src/draggable/index.js

```diff
@@ -12,7 +12,7 @@ import { withSafeTimeout } from '@wordpress/compose';
 const dragImageClass = 'components-draggable__invisible-drag-image';
 const cloneWrapperClass = 'components-draggable__clone';
 const cloneHeightTransformationBreakpoint = 700;
-const clonePadding = 20;
```

Comment:
These 20 appear to have been picked at random, back in the day, and almost certainly due to there being side UI and margins. Because there is neither anymore, I've now set this to zero, instead of cmopensating for it.